README.md

```diff
@@ -85,6 +86,23 @@ will turn on the `no-unnamed-features` rule.
 
 Any tag not included in this list won't be allowed.
 
+### file-name
+
+`file-name` is configured with a style to enforce. The default is `TitleCase`:
+
+```json
+{
+  "file-name": ["on", {"style": "TitleCase"}]
+}
+```
+
+The list of supported styles is:
+
+- `TitleCase`
+- `camelCase`
+- `kebab-case`
+- `snake_case`
+
```

Comment:
Do you mind adding a short description of what each style means?

Reply:
Sure - done